plugins/services/src/js/reducers/serviceForm/JSONReducers/Containers.js

```diff
@@ -287,6 +295,17 @@ function containersParser(state) {
   }, []);
 }
 
+function deleteContainerImageCheck(image) {
```

Comment:
I think this is awkwardly named. Maybe `shouldDeleteContainerImage` will read better.